### PR TITLE
Support arbitrary AnarchyGovernor roles

### PR DESCRIPTION
**Pull request by jkupferer**
Rather can add code in agnosticv-operator to continue to explicitly describe each AnarchyGovernor ansible role, this PR changes the operator to work with a list of roles where each role is called for each event, action, or action callback. It also adds configuration of AgnosticVRepos to the Helm chart and removes OpenShift 3 support.

### CODE REVIEW

File: roles/agnosticv/templates/governor.yaml.j2 [hunk 1]
```diff
@@ -68,84 +50,30 @@ spec:
...
-{%   if merged_vars.__meta__.aws_sandboxed | default(false) %}
-      - role: babylon_aws_sandbox
-{%   endif %}
-{% if merged_vars.__meta__.scheduler.enable | default(false) %}
-      - role: babylon_agnostics
-{% endif %}
+{%   for __role in babylon_anarchy_roles %}
+      - role: {{ __role.name }}
+{%   endfor %}
       - role: babylon_anarchy_governor
```
Comment (fridim) [inline roles/agnosticv/templates/governor.yaml.j2:68]: why is this line needed ?
it's already included with babylon_anarchy_roles, isn't it ?

Comment (jkupferer) [reply roles/agnosticv/templates/governor.yaml.j2:68]: Ah, good catch!

**Review by fridim — APPROVED** [review]
